mackup/applications/keepingyouawake.cfg

```diff
@@ -0,0 +1,5 @@
+[application]
+name = KeepingYouAwake
+
+[configuration_files]
+Library/Preferences/info.marcel-dierkes.KeepingYouAwake.plist
```

Comment:
Hi, Laurent,

Sure! Just made the change.

Do I need to issue another pull request? This was actually my first on GitHub!

Thanks for your work on this project,
-Zac

> On Aug 3, 2016, at 6:15 PM, Laurent Raufaste <notifications@github.com> wrote:
> 
> In mackup/applications/keepingyouawake.cfg <https://github.com/lra/mackup/pull/856#discussion_r73449382>:
> 
> > @@ -0,0 +1,5 @@
> > +[application]
> > +name = KeepingYouAwake
> > +
> > +[configuration_files]
> > +Library/Preferences/info.marcel-dierkes.KeepingYouAwake.plist
> Can you end this line with a newline character?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/lra/mackup/pull/856/files/d8652f60ac48df3f160b5345052ae16c2da94129#r73449382>, or mute the thread <https://github.com/notifications/unsubscribe-auth/ABopXICKO2CGBnuIqUkURLtin0oQ9nWyks5qcT0-gaJpZM4Jb-N2>.
>